test/integration/standup.test.js

```diff
@@ -0,0 +1,76 @@
+const chai = require("chai");
+const app = require("../../server");
+const addUser = require("../utils/addUser");
+const userData = require("../fixtures/user/user")();
+const authService = require("../../services/authService");
+const cleanDb = require("../utils/cleanDb");
+const { object } = require("joi");
+const { expect } = chai;
+
+const cookieName = config.get("userToken.cookieName");
+const superUser = userData[4];
+
+// eslint-disable-next-line mocha/no-skipped-tests
+describe.skip("Test standup api", function () {
```

Comment:
skip here?

Reply:
I think skip is added to prevent github warning as he is sending tests first.

Reply:
yes